include/seqan3/search/configuration/all.hpp

```diff
@@ -52,19 +57,46 @@
- * | \ref seqan3::search_cfg::max_error  "0: Max error"                  |  ❌   |  ❌   |  ✅   |  ✅   |  ✅   |
- * | \ref seqan3::search_cfg::max_error_rate "1: Max error rate"         |  ❌   |  ❌   |  ✅   |  ✅   |  ✅   |
- * | \ref seqan3::search_cfg::output "2: Output"                         |  ✅   |  ✅   |  ❌   |  ✅   |  ✅   |
- * | \ref search_configuration_subsection_hit_strategy "3. Hit"          |  ✅   |  ✅   |  ✅   |  ❌   |  ✅   |
- * | \ref seqan3::search_cfg::parallel "4: Parallel"                     |  ✅   |  ✅   |  ✅   |  ✅   |  ❌   |
+ * | **Configuration group**                                                     | **0** | **1** | **2** | **3** | **4** | **5** | **6** |
+ * | ----------------------------------------------------------------------------|-------|-------|-------|-------|-------|-------|-------|
+ * | \ref seqan3::search_cfg::max_error_total  "0: Max error total"              |  ❌   |  ✅   |  ✅   |  ✅   |  ✅   |  ✅   |  ✅   |
+ * | \ref seqan3::search_cfg::max_error_substitution "1: Max error substitution" |  ✅   |  ❌   |  ✅   |  ✅   |  ✅   |  ✅   |  ✅   |
+ * | \ref seqan3::search_cfg::max_error_insertion "2: Max error insertion"       |  ✅   |  ✅   |  ❌   |  ✅   |  ✅   |  ✅   |  ✅   |
+ * | \ref seqan3::search_cfg::max_error_deletion "3: Max error deletion"         |  ✅   |  ✅   |  ✅   |  ❌   |  ✅   |  ✅   |  ✅   |
+ * | \ref seqan3::search_cfg::output "4: Output"                                 |  ✅   |  ✅   |  ✅   |  ✅   |  ❌   |  ✅   |  ✅   |
+ * | \ref search_configuration_subsection_hit_strategy "5: Hit"                  |  ✅   |  ✅   |  ✅   |  ✅   |  ✅   |  ❌   |  ✅   |
+ * | \ref seqan3::search_cfg::parallel "6: Parallel"                             |  ✅   |  ✅   |  ✅   |  ✅   |  ✅   |  ✅   |  ❌   |
  *
  * \subsection search_configuration_search_result Search result type
  *
  * \copydetails seqan3::search_result
  *
- * \subsection search_configuration_subsection_hit_strategy 3. Hit Configuration
+ * \subsection search_configuration_subsection_error 0 - 3: Max Error Configuration
+ *
+ * This configuration can be used to specify the number or rate of error types.
+ * It restricts the number of substitutions, insertions, deletions and total errors within the search to the given
+ * values. A mismatch corresponds to diverging bases between text and query for a certain position. An insertion
+ * corresponds to a base inserted into the query that does not occur in the text at the position. A deletion corresponds
+ * to a base deleted from the query sequence that does occur in the indexed text. Deletions at the beginning and at the
+ * end of the sequence are not considered during a search.
+ * They will behave as follows:
+ * If seqan3::search_cfg::total and any other error type are specified, all types are set to the respective values.
+ * If one or more other error types are configured, but no total, total is set to the sum of the error types.
```

Comment:
I would add a new line before `They will behave as follows`:

```suggestion
 * end of the sequence are not considered during a search.
 *
 * They will behave as follows:
 * If seqan3::search_cfg::total and any other error type are specified, all types are set to the respective values.
 * If one or more other error types are configured, but no total, total is set to the sum of the error types.
```